src/vs/base/common/network.ts

```diff
@@ -113,6 +113,11 @@ export namespace Schemas {
 	 */
 	export const vscodeSourceControl = 'vscode-scm';
 
+	/**
+	 * Scheme used for input box for creating comments.
+	 */
+	export const commentsInput = 'vscode-comments-input';
```

Comment:
GHPR relies on the scheme being `comment`. Of course we can easily update GHPR, but I expect that other extensions also expect `comment`. Can we keep this as `comment`?